source/iostreams.tex

```diff
@@ -14942,7 +14942,7 @@
 \pnum
 \begin{note}
 If the directory structure being iterated over contains cycles
-then the end iterator might be unreachable.
+then it is possible for the end iterator to be unreachable.
```

Comment:
I might say, slightly more directly, "then it is possible that the end iterator is unreachable".

Reply:
That's nice, thanks. That's exactly the kind of comment I am looking for: we don't just want to change the bad words away mechanically, but ideally find nice ways to say something equivalent that avoids them.